### DISCUSSION TITLE
Adafruit fontconvert utility

### DISCUSSION THREAD

**Discussion by agganitk** [board707/DMD_STM32#50] [Q&A]
"Using Adafruit fontconvert utility allows users to convert and display on DMD matrix Truetype fonts, including Unicode fonts with national characters of almost any language."

Please share some more details about this. How to do this ?

**Comment by board707** [comment]
Hi
You can find some info about Adafruit fonts and creating new ones with `fontconert` utility on the[ link](https://learn.adafruit.com/adafruit-gfx-graphics-library/using-fonts)
